data/2011_RIVIERPRIK/processed/eml.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would organize the data similarly to bird-tracking:

    data/processed/DATASET_NAME/eml.xml

I would not store the raw eml, since we can get that one directly from IMIS.